libs/openchallenges/challenge/src/lib/challenge-overview/challenge-overview.component.html

```diff
@@ -26,10 +26,10 @@ <h3 class="section-title">Challenge Details</h3>
         <td class="text-right">Website URL</td>
         <td>{{ challenge.websiteUrl }}</td>
       </tr> -->
-      <!-- <tr>
+      <tr>
         <td class="text-right">DOI</td>
-        <td>{{ mockDoi }}</td>
-      </tr> -->
+        <td><a target="_blank" href="#">{{ mockDoi }}</a></td>
```

Comment:
Replace mock by actual DOI link?

Reply:
I agree, replacing mock DOI by actual DOI can be do in another PR.

Reply:
Great! I added it as a Story to our June sprint.